src/auth/webhook-secret.guard.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  CanActivate,
+  ExecutionContext,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
+import * as crypto from 'crypto';
+import { Request } from 'express';
```

Comment:
PullProofAI

Consider adding type checking for the `Request` object to improve code clarity and maintainability.
```diff
+ import { Request } from 'express';
```

#### Issue: Unused Imports


<!-- This is an auto-generated comment by PullProofAI -->